jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/assertions/AttributeMatcher.java

```diff
@@ -76,7 +58,7 @@ public String toString() {
    * @return true if this matcher is matching provided value, false otherwise.
    */
   boolean matchesValue(String value) {
-    if ((attributeValue == null) || (value == null)) {
+    if (attributeValue == null) {
```

Comment:
here the value is never `null` as there are no attribute with `null` value in otel.

Reply:
Right, I think the updated code does the same as original one, just restructured to avoid using short circuit?